### PR TITLE
refactor: tag component to work with button/link/checkbox/radio

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

**Review by mfranzke — DISMISSED** [review]

**Comment by mfranzke** [comment]
@leape I think to remember that the tag component is even also supposed to function as a link, meaning that it could link to another page, isn't it?

File: packages/components/src/components/brand/brand.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@
 }
 
 .db-logo {
+	max-width: none;
```
[the remark this answers is not in the document]
Comment (nmerget) [reply packages/components/src/components/brand/brand.scss:22]: For the header. If we have a lot of navigation items, the logo shrinks with max-width:100%, which is set by `_init.scss` https://github.com/db-ui/mono/blob/main/packages/foundations/scss/_init.scss#L63

**Comment by nmerget** [comment]
> @leape I think to remember that the tag component is even also supposed to function as a link, meaning that it could link to another page, isn't it?

I changed the tag. We pass the correct component as child, the tag is basically just a styling wrapper.

**Comment by nmerget** [comment]
Pipeline is not restarting, created a new PR https://github.com/db-ui/mono/pull/1288

**Pull request closed without merging**